velocidi-sdk/src/main/kotlin/com/velocidi/HttpClient.kt

```diff
@@ -1,32 +1,24 @@
 package com.velocidi
 
+import android.net.Uri
 import android.util.Log
-import com.android.volley.RequestQueue
-import com.android.volley.Response
-import com.android.volley.Response.Listener
-import com.android.volley.toolbox.BasicNetwork
-import com.android.volley.toolbox.DiskBasedCache
-import com.android.volley.toolbox.HurlStack
-import java.io.File
-import com.android.volley.toolbox.StringRequest
 import org.json.JSONObject
 import java.lang.Exception
-import java.net.URL
-import java.nio.charset.Charset
 import com.velocidi.Util.appendToUrl
+import okhttp3.*
+import okhttp3.Request
+import java.io.IOException
 
 /**
  * Http Client based on Android Volley
  *
  */
-internal class HttpClient {
-    private val cache = DiskBasedCache(File(Constants.CACHE_DIR), 1024 * 1024) // 1MB cap
```

Comment:
This `CACHE_DIR` constant is no longer required it seems.

Reply:
Removed in 211a8b1